shm/cargo.hpp

```diff
@@ -2,15 +2,19 @@
 #include <string>
 class Cargo {
 public:
+    Cargo(std::string name, size_t amount, size_t basePrice);
     Cargo& operator+=(size_t amount);
     Cargo& operator-=(size_t amount);
     bool operator==(std::string name);
+    std::string GetName() const;
+    size_t GetAmount() const;
+    size_t GetBasePrice() const;
 
 
 
```

Comment:
too many empty lines. One line is enough

Reply:
we can autoformat it later, he uses vim

Reply:
ok :)